chart/flux/values.yaml

```diff
@@ -203,7 +203,7 @@ memcached:
   enabled: true
   hostnameOverride:
   repository: memcached
-  tag: 1.5.20
+  tag: 1.6.9-alpine
```

Comment:
Having doubts about combining a MINOR version update of a dependency with a PATCH update of the chart itself.

Reply:
If you think it makes sense, I will amend the chart bump and call it 1.8.0

Reply:
(Done)